pkgs/servers/http/nginx/modules.nix

```diff
@@ -311,6 +312,14 @@ let self = {
     supports = with lib.versions; version: major version == "1" && minor version == "19";
```

Comment:
Should we drop the old ones?

Reply:
Sounds reasonable. Added a commit removing old broken modules.